### PR TITLE
Update vimr to v0.9.0-SNAPSHOT-20161017.2152-107

### PR DESCRIPTION
**Pull request by pbdeuchler**
After making all changes to the cask:
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] The commit message includes the cask’s name and version.

### CODE REVIEW

File: Casks/vimr.rb [hunk 1]
```diff
@@ -1,11 +1,12 @@
 cask 'vimr' do
-  version '0.8.0-32'
-  sha256 '8660e7d5157b3165faab8a228f67a1bd63016c51bc300af131b90edc18dda678'
+  version 'v0.9.0-SNAPSHOT-20161017.2152-107'
```
Comment (vitorgalvao) [inline Casks/vimr.rb]: Leave the `v` in the `url`.

File: Casks/vimr.rb [hunk 2]
```diff
@@ -1,11 +1,12 @@
...
+  sha256 '0ef84f2f2dddce3d0a4ab1c7578bf28d3ecd24d12e9554b7cac610974e4af447'
 
   # github.com/qvacua/vimr was verified as official when first introduced to the cask
-  url "https://github.com/qvacua/vimr/releases/download/v#{version}/VimR-#{version.sub(%r{-.*}, '')}.tar.bz2"
-  appcast 'https://github.com/qvacua/vimr/releases.atom',
-          checkpoint: 'cfd99e9ff5feb321bde77647d980df524d1522c8203df977f7a245254020c250'
+  url "https://github.com/qvacua/vimr/releases/download/snapshot%2f#{version}/VimR-#{version}.tar.bz2"
+  appcast 'https://raw.githubusercontent.com/qvacua/vimr/master/appcast.xml',
+          checkpoint: 'ee806d0b9294d6a886dc659f951a250fe3d4bbfcb75d919a8cc7768d922dd1b4'
+  auto_updates true
```
Comment (vitorgalvao) [inline Casks/vimr.rb]: This is in the wrong place. `brew cask style --fix` should fix that. Did you run that?

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

**Comment by pbdeuchler** [comment]
@vitorgalvao first time contributor, sorry about that! should be all fixed

File: Casks/vimr.rb [hunk 3]
```diff
@@ -1,14 +1,16 @@
...
-  sha256 '8660e7d5157b3165faab8a228f67a1bd63016c51bc300af131b90edc18dda678'
+  version '0.9.0-SNAPSHOT-20161017.2152-107'
+  sha256 '0ef84f2f2dddce3d0a4ab1c7578bf28d3ecd24d12e9554b7cac610974e4af447'
 
   # github.com/qvacua/vimr was verified as official when first introduced to the cask
-  url "https://github.com/qvacua/vimr/releases/download/v#{version}/VimR-#{version.sub(%r{-.*}, '')}.tar.bz2"
-  appcast 'https://github.com/qvacua/vimr/releases.atom',
-          checkpoint: 'cfd99e9ff5feb321bde77647d980df524d1522c8203df977f7a245254020c250'
+  url "https://github.com/qvacua/vimr/releases/download/snapshot%2fv#{version}/VimR-#v{version}.tar.bz2"
+  appcast 'https://raw.githubusercontent.com/qvacua/vimr/master/appcast.xml',
```
Comment (vitorgalvao) [inline Casks/vimr.rb]: Why change this? We prefer the other one.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/vimr.rb (hunk 3 above)
Comment (pbdeuchler) [inline Casks/vimr.rb]: Works for me, updated

**Review by vitorgalvao — APPROVED** [review]

**Comment by vitorgalvao** [comment]
Thank you for the contribution and quick fixes! As soon as Travis passes, we can merge this.

**Comment by pbdeuchler** [comment]
Happy to help!

**Comment by jawshooah** [comment]
Wait, why are we upgrading to a snapshot release in the first place? It seems like there are [multiple snapshots every day](https://github.com/qvacua/vimr/releases), and they are all labeled "Pre-release". I feel like this belongs in the versions repo as `vimr-snapshot`.

**Comment by pbdeuchler** [comment]
I wouldn't say every day, but no denying there are frequent releases. However this cask update fetches the latest version that includes Sparkle (plus a minor bugfix), so now anyone who installs will be on a better upgrade path for the future. However installing the snapshot doesn't force you into the nightly channel, so there will still be a stable experience (as stable as pre 1.0 gets at least). 

Most importantly though, anyone who would be installing vimr for the first time would use this release (as of today) if they followed the project README, while the current cask still installs a version that's almost a year old.

**Comment by jawshooah** [comment]
@pbdeuchler The appcast checksum is now inaccurate because there has been another snapshot release since this PR was created. Which touches on my concern: which such frequent snapshot releases, how will this cask be kept up-to-date? Is there not a series of stable releases? You say that installing a snapshot release doesn't force you into a nightly channel, but what other channel is there?

**Comment by fanquake** [comment]
@pbdeuchler Can you follow up here?

**Comment by pbdeuchler** [comment]
Apologies, I must have accidentally inbox zeroed the previous notification email.

I wouldn't say the latest release changes my opinion at all, I'd like to re-iterate that the only releases for over the past year have been snapshot releases, and given that the official README of the project recommends downloading the latest bleeding edge release I don't necessarily agree that a separate cask is needed. 

As far as separate update channels are concerned, I am not on the nightly Sparkle channel and have yet to have an update come down the wire. I am assuming the next push down the default channel would be a major version however I think the maintainer would probably be the best person to ask about that.

Just to expedite this conversation, is the sticking point that this PR moves the version to a non-major release, or that the app is updated too frequently to warrant a cask not named with some sort of warning (`snapshot`, `nightly`, etc)?

**Comment by jawshooah** [comment]
@pbdeuchler Thanks for getting back. Looking at the description of the [most recent snapshot release](https://github.com/qvacua/vimr/releases/tag/snapshot%2Fv0.9.0-SNAPSHOT-20161020.2036-111), it seems this discussion might be moot:

> Probably the last snapshot before the 0.9.0 release

Since a new stable release is coming soon, I think I'd rather wait. In the meantime, this can be added to [homebrew-versions](https://github.com/caskroom/homebrew-versions) as `vimr-snapshot`.

**Comment by pbdeuchler** [comment]
Works for me, I'll just wait and submit the stable version

**Comment by jawshooah** [comment]
@pbdeuchler https://github.com/qvacua/vimr/releases/tag/v0.9.0-112

**Comment by pbdeuchler** [comment]
Woohoo! You're on top of it @jawshooah... submitted here: https://github.com/caskroom/homebrew-cask/pull/26016

**Pull request closed without merging**